### ISSUE TITLE
Consider cross-references to other terms associated with #cwrcLabels

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/ontology#221]
For definitions such as the following:

#jewishNationalIdentity
Indicates identification with or labelling as Jewish as a national identity.

It would be nice to include links to the other terms associated with #jewishLabel, e.g. to say:

Related to these other terms associated with #jewishLabel: jewishReligion; jewishEthnicity

@alliyya do you think these could be generated by a script so that if more terms were added, the list could automatically get augmented? (this is NOT for the next release, but am just recording it now)

**Comment by SusanBrown** [comment]
Also @alliyya I believe we agreed that we could list all terms related to labels. So for instance right now englishLabel has a skos:broader link to the LOD language code, but we will add englishNationality etc. I am thinking, as I believe we discussed, that this would be sufficient and that we wouldn't need to x-reference between those narrower terms themselves. 

If you agree, then let's proceed to do this.  Do the linkages need to be done manually?

**Comment by SusanBrown** [comment]
So, as decided in our meeting, these will now use the predicate represents. I've added this to the spreadsheets to be documented before being added to the ontology.

**Comment by SusanBrown** [comment]
I believe that this is done.

**Comment by alliyya** [comment]
@SusanBrown 
I thought you had wanted this programmatically done so that the definitions for ex. jewishEthnicity would include reference/links to cwrc:jewishGeographicHeritage cwrc:jewishNationalIdentity cwrc:jewishRaceColour cwrc:judaism
 from the shared jewishLabel?

**Comment by SusanBrown** [comment]
Yes, if we can do it programmatically that would be great. So maybe this is not done. Or we should create another issue. I thought this was about the represents predicate but it was about two things, I think

**Comment by alliyya** [comment]
I think there was separate issue for represents predicates, reopening this one.

**Comment by alliyya** [comment]
Was this to add to the definition of the specific instance or the textual label itself.
ex.
#jewishNationalIdentity
Indicates identification with or labelling as Jewish as a national identity. _Related to these other terms associated with jewishLabel: judaism; jewishEthnicity._

or 
#jewishLabel 
A subclass of @@#TextLabels@@, this discursive label reflects the ambiguity of Jewishness associated with different @@#CulturalForm@@s. It provides a means of aggregating and searching multiple instances of "Jewish" (e.g. @@#jewishNationalIdentity@@, @@#jewishRaceColour@@) cultural identities. _Related to these other terms associated with jewishLabel: judaism; jewishEthnicity._

**Comment by SusanBrown** [comment]
The first but ideally also the second. Do the first first. 

Line break before italicized text for both. The terms will be linked, correct?
Second could just read “Terms associated with ...,”

For second

**Comment by alliyya** [comment]
Or should this be included in an rdfs:comment rather than the skos:definition for those terms. 

Terms will be linked, should the text however be the label for the term or the actual URI?

**Comment by SusanBrown** [comment]
Let’s try using the labels for now. 

Not sure if they would be better in comments or definitions. Consult w Joel and Deb? I am on phone and can’t tag them. I feel the definition includes the relationships and so lean towards inclusion.
 
Are we creating skos:related between the terms as well?

**Comment by alliyya** [comment]
The only issue with doing it with the labels, they tend to be identical so you'd end up with 

`Terms associated with Jewish, Jewish, Jewish, Judaism.`  

Do we want those skos:related added programmatically in the final version or put in manually as of now?
